packages/app/src/Charcoal/App/Facade/Facade.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+namespace Charcoal\App\Facade;
+
+use Charcoal\App\App;
+use RuntimeException;
+
+/**
+ * Facade
+ *
+ * The facade class acts as a shortcut to a container service.
+ */
+class Facade
+{
+    protected static App $app;
+    /**
+     * @var array<string, object>
+     */
+    protected static array $resolvedInstances = [];
+
+    public static function setFacadeApp(App $app)
+    {
+        static::$app = $app;
+    }
```

Comment:
I think the Facade should only have access to the Container (would be called a `facadeContainer` or `facadeResolver`). I don't think the Facade should be aware of the Application.

In Laravel, the Application _is_ the Container where as in Charcoal the Application and Container are separate.

Reply:
Done